charts/orchestrator/templates/sonataflows.yaml

```diff
@@ -1,22 +1,38 @@
-{{- $ns := .Values.orchestrator.namespace }}
-{{- if .Values.orchestrator.devmode }}
-  {{- $namespace := (lookup "v1" "Namespace" "" $ns) }}
-  {{- if $namespace }}
-# Namespace exist, do nothing.
-  {{- else }}
+{{- define "persistence" }}
+      persistence:
```

Comment:
Unrelated to this PR, but how do we pass persistence configuration to the workflows? 
Is it enough to have this section configured in the sonataflowplaform CR to inherit it to the workflows?

Reply:
Yes. By default workflows will inherit this configuration unless they defined their own persistence in their CR.